### ISSUE TITLE
Accessibility Issues

### ISSUE THREAD

**Issue by Emmarie-Ahtunan** [teamhanko/hanko#1698]
### Checklist

- [X] I could not find a solution in the existing issues or docs.
- [X] I agree to follow this project's [Code of Conduct](https://github.com/teamhanko/hanko/blob/main/CODE_OF_CONDUCT.md).

### Describe the bug

- Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds

[more information. Link opens in a new window](https://dequeuniversity.com/rules/axe/4.10/color-contrast?application=AxeChrome)
Element Location:
.witdh-200px > .button-text

To solve this problem, you need to fix the following:
Element has insufficient color contrast of 3.65 (foreground color: #ffffff, background color: #ff2e4c, font size: 12.8pt (17px), font weight: normal). Expected contrast ratio of 4.5:1

Related Node
<a href="https://cloud.hanko.io" class="button-primary witdh-200px is-centered w-inline-block"><div class="button-text"></a>

- Ensure <img> elements have alternate text or a role of none or presentation

[more informationLink opens in a new window](https://dequeuniversity.com/rules/axe/4.10/image-alt?application=AxeChrome)
Element Location:
.html-embed-4.w-embed > img
<img src="https://hanko.openstatus.dev/badge">
To solve this problem, you need to fix at least (1) of the following:
Element does not have an alt attribute
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
Element has no title attribute
Element's default semantics were not overridden with role="none" or role="presentation"
Found:
Automatically
Impact:
critical
cat.text-alternatives
wcag2a
wcag111
section508
section508.22.a
TTv5
TT7.a
TT7.b
EN-301-549
EN-9.1.1.1
ACT
Found on:
10/12/2024 at 10:06 PM

![Image of Hanko Accessibility Scan](https://github.com/user-attachments/assets/9a8e60ab-756f-4bf2-896e-1628999c814b)

### Reproducing the bug

not applicable.

### Hanko Version

1.0.2

### OS Hanko Backend

macOS

### OS

None

### Environment

None
